torch/distributed/distributed_c10d.py

```diff
@@ -1047,9 +982,32 @@ def _new_process_group_helper(
                 dist_backend_opts.global_ranks_in_group = global_ranks_in_group
 
                 pg = creator_fn(dist_backend_opts, pg_options)
-            _world.pg_map[pg] = (backend, store)
-            _world.pg_names[pg] = group_name
 
+    # Process group wrapper initialization for supported PGs when TORCH_DISTRIBUTED_DEBUG is set
+    if backend in [Backend.GLOO, Backend.NCCL, Backend.UCC]:
+        # In debug mode and if GLOO is available, wrap in a wrapper PG that
+        # enables enhanced collective checking for debugability.
```

Comment:
nit: 
```suggestion
        # enables enhanced collective checking for debuggability.
```
https://wikidiff.com/debugability/debuggability

Reply:
thank you sir